app/assets/stylesheets/zammad.scss

```diff
@@ -8064,6 +8064,11 @@ output {
 .searchableSelect {
   position: relative;
 
+  .selected_organization {
```

Comment:
What is required for? There should be no hard coded attribute logic in generic files (selected_organization).